solr/core/src/test/org/apache/solr/cloud/ZkSolrClientTest.java

```diff
@@ -205,14 +205,14 @@ public void testZkCmdExectutor() throws Exception {
       try {
       zkCmdExecutor.retryOperation(() -> {
         if (System.nanoTime() - start > TimeUnit.NANOSECONDS.convert(timeout, TimeUnit.MILLISECONDS)) {
-          throw new KeeperException.SessionExpiredException();
```

Comment:
My understanding was that SessionExpired was just being used as an alternate (non-caught) exception type, SessionExpired itself wasn't really being tested here, just something besides ConnectionLoss.  So I believe the intent of this test was to check:
1) that caught exceptions are retried
2) that non-caught exceptions are passed through.

so i picked a new exception type for 2) that preserves the checks of this test.